src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/PackageUpdaterStep.cs

```diff
@@ -32,7 +32,7 @@ public class PackageUpdaterStep : UpgradeStep
 
         public override string Description => "Update package references to versions compatible with the target framework";
 
-        public override string Title => "Update NuGet packages";
+        public override string Title => "Update Nuget Packages";
```

Comment:
I think NuGet is the proper casing, but we should at least be consistent with out stuff

Reply:
Fixed